pymemcache/client/base.py

```diff
@@ -1039,6 +1039,9 @@ class PooledClient(object):
       max_pool_size: maximum pool size to use (going above this amount
                      triggers a runtime error), by default this is 2147483648L
                      when not provided (or none).
+      pool_idle_timeout: pooled connections are discarded if they have been
+                         unused for this many seconds. A value of 0 or None
+                         indicates that pooled connections are never discarded.
```

Comment:
Given that `0` is as safe of a "truthy" sentinel value as `None` (and we upgrade `None` to `0` internally), I think we should always say this value is an `int` that defaults to `0`. What do you think?

Reply:
Sure, I'm fine with that. Should `max_pool_size` be handled the same way?

Reply:
Probably, but we've already shipped `max_pool_size` in public releases, so it's best to leave it be to avoid accidentally breaking existing installations.